src/content/es/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -712,62 +718,62 @@ evt.respondWith(
 );
 ```
 
-El código intercepta la solicitud y verifica si es para un pronóstico del tiempo. Si es así, use `fetch` para realizar la solicitud. Una vez que se devuelve la respuesta, abra la memoria caché, clone la respuesta, almacénela en la memoria caché y devuelva la respuesta al solicitante original.
+El código intercepta la solicitud y verifica si es para un pronóstico del tiempo. Si es así, usa `fetch` para realizar la solicitud. Una vez que se devuelve la respuesta, abre la memoria caché, clona la respuesta, almacénela en la memoria caché y devuelve la respuesta al solicitante original.
 
-Necesitamos eliminar la comprobación `evt.request.mode !== 'navigate'` porque queremos que nuestro trabajador de servicio maneje todas las solicitudes (incluidas imágenes, scripts, archivos CSS, etc.), no solo las navegaciones. Si dejamos ese registro, solo se entregará el HTML desde la memoria caché del trabajador de servicio, todo lo demás se solicitará desde la red.
+Necesitamos eliminar la comprobación `evt.request.mode !== 'navigate'` porque queremos que nuestro service worker maneje todas las solicitudes (incluidas imágenes, scripts, archivos CSS, etc.), no solo las navegaciones. Si dejamos ese registro, solo se entregará el HTML desde la memoria caché del service worker, todo lo demás se solicitará desde la red.
 
 ### Pruébalo
 
-La aplicación debe estar completamente fuera de línea funcional ahora. Actualice la página para asegurarse de que tiene instalado el trabajador de servicio más reciente, luego guarde un par de ciudades y presione el botón de actualización en la aplicación para obtener información actualizada sobre el clima.
+La aplicación debe estar completamente sin conexión funcional ahora. Actualiza la página para asegurarte de que tienes instalado el service worker más reciente, luego guarda un par de ciudades y presione el botón de actualización en la aplicación para obtener información actualizada sobre el clima.
 
-Luego vaya al panel __Cache Storage__ en el panel __Aplicación__ de DevTools. Expanda la sección y verá el nombre de su caché estático y el caché de datos en el lado izquierdo. Al abrir el caché de datos se deben mostrar los datos almacenados para cada ciudad.
+Luego ve al panel __Cache Storage__ en el panel __Application__ de DevTools. Expanda la sección y verá el nombre de su caché estático y la caché de datos en el lado izquierdo. Al abrir la caché de datos se deben mostrar los datos almacenados para cada ciudad.
 
 ![731e91776cb6ef18.png](img/731e91776cb6ef18.png)
 
-Luego, abra DevTools y cambie al panel Trabajadores de servicio, y marque la casilla de verificación Sin conexión, luego intente volver a cargar la página, luego desconecte y vuelva a cargar la página.
+Luego, abre DevTools y cambia al panel service workers, y marca la casilla de verificación Offline, luego intente volver a cargar la página, después desconecta y vuelve a cargar la página.
```

Comment:
`luego intenta volver a cargar la página` to keep the conjugation in the second person.